packages/DataTable/src/components/InfiniteLoader/InfiniteLoader.tsx

```diff
@@ -92,6 +94,7 @@ export function InfiniteLoaderImpl({
           onItemsRendered={onItemsRendered}
           ref={listRef}
           innerElementType={innerElementType}
+          style={{ overflowX: shouldHaveHorizontalScroll ? "auto" : "hidden" }}
```

Comment:
When there's a vertical scrollbar but the horizontal scrollbar is not needed, the horizontal scrollbar is likely still there since the vertical one takes some horizontal space. We either:
1. make the `best width` bigger, first we need to calculate the scrollbar width, second this might cause the table has a bigger width than the `width` prop
2. set an `overflow-x: hidden`, then the last column might be cut off if it's too narrow, but I think a very narrow column in the last column is kind of an edge case, so I went with this 
3. just don't do anything 😂

Reply:
#1 sounds potentially better, but more complicated, so I agree with #2 for now.  If we were to change later to #1, would it have any impact on the API?

Reply:
> would it have any impact on the API


I don't think so, it's an internal calculation anyway

Reply:
Ticket created: https://aclgrc.atlassian.net/browse/UXD-1968